cfme/networks/views.py

```diff
@@ -33,7 +33,7 @@ class NetworkProviderToolBar(View):
 class NetworkProviderDetailsToolBar(NetworkProviderToolBar):
     """ Represents provider details toolbar """
     monitoring = Dropdown(text='Monitoring')
-    download = Button(title='Download summary in PDF format')
+    download = Button(title='Print or export summary')
```

Comment:
@quarckster Hello Dima, please can you tell me how to use version.picker here?
I need to keep the old text for 5.9 backwards compatibility.
Thanks

Reply:
```python
from cfme.utils.version import VersionPicker, Version
...
download = VersionPicker(
  Version.LOWEST: Button(title='Print or export summary'),
  "5.10": Button(title='New title')
)
```